var/spack/repos/builtin/packages/powerapi/package.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Powerapi(AutotoolsPackage):
+    """This software is a reference implementation of the PowerAPI"""
+
+    homepage = "https://powerapi.sandia.gov/"
+    git      = "https://github.com/pwrapi/pwrapi-ref/tree/v1.1.1.git"
```

Comment:
This git URL does not work for me but `https://github.com/pwrapi/pwrapi-ref.git` does. The commit specification in the `version` declaration will ensure that the appropriate commit is retrieved.

Reply:
Oh ok, I submitted a change that should fix it.